src/common/loggingEvents.ts

```diff
@@ -8,23 +8,54 @@ export interface BaseEvent {
     type: EventType;
 }
 
+export enum SignResult {
+    Succeeded,
+    Failed

```

Comment:
if there's only 'Succeeded' and 'Failed', what about simply using boolean field in `UserSignInCompleted`. e.g. `succeeded`?